primer/brand
# Button group component

## Summary

Creating a ButtonGroup component that will handle the positioning and styling of buttons to be used in the CTABanner and Hero components

## List of notable changes:

- **created** ButtonGroup component
- **added** story to Storybook documentation

## What should reviewers focus on?

- The functionality and robustness of the component

## Steps to test:

1. Go to ButtonGroup story in Storybook
2. Verify that the ButtonGroup works as expected
3. Create a demo site and import the ButtonGroup
4. Try putting in both expected and not expected items within it

## Supporting resources (related issues, external links, etc):

- [Figma design](https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=1020%3A29442)
- [Storybook documentation](https://primer-f4e099c77f-26139705.drafts.github.io/storybook/?path=/story/components-buttongroup--primary)

## Contributor checklist:


## Reviewer checklist:
